apis/github

```diff
@@ -103,15 +103,15 @@ Repository.cloneTo = function(self, dest, onProgress)
 		return error("Destination is a file!")
 	end
 
-	for item in self:tree('master'):iter() do
+	for item in self:tree(sha):iter() do
```

Comment:
You've just made the api not work unless someone defines a global `sha` variable.